uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-popup/details-tab/by-unit-and-days-mode/days-unit-single-day.html

```diff
@@ -0,0 +1,3 @@
+<ng-include src="$root.sharedPathTpl +
```

Comment:
why not get rid of this partial and include `days-unit-from-date-and-type.html` directly wherever you need it? 

If you have this file for the convenience of the name rename `days-unit-from-date-and-type.html` to `days-unit-single-day.html` and use that one elsewhere.

Reply:
That partial is used in 2 files, also in `days-unit-multiple-days.html`